### DISCUSSION TITLE
[Closed] Changing Group For Clients

### DISCUSSION THREAD

**Discussion by tyl0re** [Ylianst/MeshCentral#4266] [Ideas]
It would be good, if it possible to change the group on  an client.
So it would be Possible, to Spit Groups, without new installation of an Client

**Comment by tyl0re** [comment]
Sorry have found it that it is possible , THought it would be possible in the Device View, havnt looked in the Groupview

**Reply by folosleg** [reply]
Both is possible! On the device view, bottom right-ish corner:
![image](https://user-images.githubusercontent.com/80481320/178955259-b85a4896-bc80-47fe-9e7b-9a0a44ef16d4.png)
